src/cli/README.md

```diff
@@ -1086,6 +1087,43 @@ Set the minimal delay timer (in seconds).
 Done
 ```
 
+### deviceconfig
+
+Get the current device config.
+
+```bash
+> deviceconfig
+PowerSupply      : external
+IsBorderRouter   : yes
+SupportsCcm      : no
+IsUnstable       : no
+WeightAdjustment : 0
+Done
+```
+
+### devicefonfig \<power-supply\> \<is-br\> \<supports-ccm\> \<is-unstable\> \<weight-adjustment\>
+
+Set the device configurations which are then used to determine and set the Leader Weight.
+
+- power-supply: 'battery', 'external', 'external-stable', or 'external-unstable'.
+
```

Comment:
Can add here a line for the weight-adjustment parameter; saying that it will be interclamped -16 ... +16. (Would be good to point this out to the CLI user, as I can set a value of +80 which is shown in 'deviceconfig', but silently clamped to 16.)

Reply:
Good idea. Added it in `README`.
Also added logic to clamp the value when we set it (if out of range) so what we read is what we apply.
```bash
> deviceconfig external 1 0 0 -20
Done

> deviceconfig
PowerSupply      : external
IsBorderRouter   : yes
SupportsCcm      : no
IsUnstable       : no
WeightAdjustment : -16
```